pkgs/applications/networking/owncloud-client/default.nix

```diff
@@ -1,34 +1,51 @@
 { lib
 , stdenv
 , fetchFromGitHub
-, mkDerivation
+, callPackage
 , pkg-config
 , cmake
 , extra-cmake-modules
-, callPackage
-, qtbase
-, qtkeychain
+, shared-mime-info
+, qt6
+, qt6Packages
 , wrapQtAppsHook
-, qttools
 , sqlite
 , libsecret
+# darwin only:
+, libinotify-kqueue
+, sparkleshare
 }:
 
 stdenv.mkDerivation rec {
   pname = "owncloud-client";
-  version = "4.2.0";
+  version = "5.0.0";
 
-  libregraph = callPackage ./libre-graph-api-cpp-qt-client.nix { };
+  # track src/libsync/CMakeLists.txt
```

Comment:
Why are those libraries not available for other applications in top-level?

Reply:
The library lives in an extra repo and is properly packaged, so it should gets it's own top-level entry.